Website/src/rendering/src/pages/_app.tsx

```diff
@@ -71,11 +79,15 @@ function App({ Component, pageProps, router }: AppProps): JSX.Element {
       */}
       <I18nProvider lngDict={dictionary} locale={pageProps.locale}>
         {/* DEMO TEAM CUSTOMIZATION - OrderCloud integration */}
-        <Provider store={reduxStore}>
-          <OcProvider>
-            <Component {...rest} />
-          </OcProvider>
-        </Provider>
+        {isCommerceEnabled ? (
```

Comment:
To simplify code in here, could the condition be moved in `OcProvider.tsx` (inside the `useEffect` maybe) and in `store.ts` (maybe building a store with a fake reducer)?

Reply:
Is the website working correctly when commerce is disabled with the current code (with no changes to the Redux store)? If the site is working correctly with no errors/warnings in the browser console nor in the rendering container logs, I would say the current solution is enough.

However, if there are still errors, we would have to find a way to fix those errors.

Reply:
Yeah the website seems to be working correctly with commerce disabled and no changes to the Redux store. No errors/ warnings in the console or the rendering container.